FlymFork/src/main/res/values/strings.xml

```diff
@@ -27,31 +27,31 @@
     <string name="about_flym">About the app / Help / Contact developer. </string>
     <string name="welcome_title">Welcome to Flym Fork. Would you like to add a feed?</string>
     <string name="never">never</string>
-    <string name="update">Update</string>
+    <string name="update">update</string>
```

Comment:
This is a caption

Reply:
I've checked where it is displayed and it is one of the verbs which inform on what is happening (on update) - those verbs which we've discussed earlier which should have the "-ing" ending. So I'll change it that way ("Updating...").

Reply:
Should I change anything and again make pull requests - or just to comment here?